modules/imgcodecs/src/grfmt_jpeg.cpp

```diff
@@ -499,7 +499,7 @@ bool  JpegDecoder::readData( Mat& img )
                 for( int iy = 0 ; iy < m_height; iy ++ )
                 {
                     uchar* data = img.ptr<uchar>(iy);
-                    jpeg_read_scanlines( cinfo, &data, 1 );
```

Comment:
https://refspecs.linuxfoundation.org/LSB_3.1.1/LSB-Desktop-generic/LSB-Desktop-generic/libjpeg.jpeg.read.scanlines.1.html It may read more.

Reply:
Indeed, I now take it into account.